DNN Platform/Library/Entities/Tabs/TabVersions/TabVersionController.cs

```diff
@@ -112,6 +119,34 @@ private static string GetTabVersionsCacheKey(int tabId)
         {
             return string.Format(DataCache.TabVersionsCacheKey, tabId);
         }
+
+        private IEnumerable<TabVersion> ApplyUserLocalTime(IEnumerable<TabVersion> tabVersions, TimeSpan userUtcOffset, TimeZoneInfo userTimeZone)
+        {
+            return tabVersions.Select((tabVersion) =>
+            {
+                var serverUtcOffset = TimeZoneInfo.Local.BaseUtcOffset;
+                if (serverUtcOffset.CompareTo(userUtcOffset) >= 0)
+                {
+                    tabVersion.CreateOnUserLocalDate = ApplyDaylightSavingHour(userTimeZone, tabVersion.CreatedOnDate.Subtract(serverUtcOffset).Add(userUtcOffset));
```

Comment:
There is also a `Timestamp` property. Which is used on the versions UI?

Reply:
Timestamp is not used in UI. I checked before to apply the fix :)